add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -186,6 +186,48 @@ const kMessageHandlers = {
   },
 
   /**
+   * Creates a layout for the remote cursor on the browser chrome,
+   * and positions it on the received coordinates.
+   *
+   * @param {Object}  message Message meant for the handler function, containing
+   *                          the following parameters in its 'data' property:
+   *                          {
+   *                            ratioX: cursor's X position (between 0-1)
+   *                            ratioY: cursor's Y position (between 0-1)
+   *                          }
+   *
+   * @param {Function} reply  Callback function, invoked with the result of the
+   *                          message handler. The result will be sent back to
+   *                          the senders' channel.
+   */
+  AddRemoteCursorOverlay: function(message, reply) {
+    let win = Services.wm.getMostRecentWindow("navigator:browser");
+    let browser = win.gBrowser.selectedBrowser;
+    let cursor = browser.parentNode.querySelector("#remote-cursor");
+
+    if (!cursor) {
+      cursor = win.document.createElement("image");
+      cursor.setAttribute("id", "remote-cursor");
+      cursor.style.pointerEvents = "none";
+    }
+
+    let actionData = message.data[0];
+    // Calculate the final position based on browser's size
+    let position = {
+      left: actionData.ratioX * browser.boxObject.width,
+      top: actionData.ratioY * browser.boxObject.height
+    };
+
+    // Update the cursor's position
```

Comment:
Full sentences for comments (add period) and mention the parent is a xul:stack (so that's why the attribute left/top work).